manifests/params.pp

```diff
@@ -68,7 +77,6 @@
         'python-psycopg2',
         'python-simplejson',
         'python-sqlite',
-        'python-twisted',
```

Comment:
twisted was being installed twice, once via the autodetected provider and once via pip, since the pip method was specifying a version I assumed this one was redundant